### DISCUSSION TITLE
Update to a newer version?

### DISCUSSION THREAD

**Discussion by BeagleNF404** [nonamecrackers2/crackers-wither-storm-mod#2032] [Q&A] [answered]
I have been intressed to add the witherstorm for a event in my server when it updates to _1.21_. Is it possible that the mod be updated to 1.21?

**Comment by nonamecrackers2** [comment, marked as answer]
Yes, it will be released when it is ready

**Reply by BeagleNF404** [reply]
Thankyou so much!
